.travis.yml

```diff
@@ -17,3 +24,6 @@ script:
 
 after_script:
     - docker-compose -f docker-compose.yaml down
+
+after_success:
```

Comment:
Looks like it's [run before `after_script`](https://docs.travis-ci.com/user/customizing-the-build/#the-build-lifecycle), would be useful to keep the same order for clarity.